.github/workflows/continuous-integration-linux.yml

```diff
@@ -19,7 +19,6 @@ jobs:
         cxx_extra_flags: ['']
         cmake_build_type: ['Release', 'Debug']
         backend: ['OPENMP']
-        clang-tidy: ['']
```

Comment:
Do you think we should exclude `g++` builds ? I don't know how reliable is the support compilers other than `clang++`.

Reply:
It seems to work now after removing `-Wclobbered` and I think we reevaluate when problems appear.

Reply:
`-Wclobbered` is included in `-Wextra` and also not really relevant for `Kokkos` AFAICT.

Reply:
I don't have a strong opinion about disabling for `g++`. It's not clear to me if the build time is significantly longer and it makes writing the CI file more difficult.